test/integration/02-user-wallet/02-send-onchain.spec.ts

```diff
@@ -195,147 +241,237 @@ describe("UserWallet - onChainPay", () => {
       displayAmount: displayPaymentAmount,
     })
 
-    expect(sendNotification.mock.calls.length).toBe(1)
     expect(sendNotification.mock.calls[0][0].title).toBe(title)
     expect(sendNotification.mock.calls[0][0].body).toBe(body)
+  }
 
-    {
-      const txResult = await Wallets.getTransactionsForWalletId({
-        walletId: walletIdA,
-      })
-      if (txResult.error instanceof Error || txResult.result === null) {
-        throw txResult.error
-      }
-      const pendingTxs = txResult.result.filter(
-        ({ status }) => status === TxStatus.Pending,
-      )
-      expect(pendingTxs.length).toBe(0)
+  {
+    const txResult = await Wallets.getTransactionsForWalletId({
+      walletId: senderWalletId,
+    })
+    if (txResult.error instanceof Error || txResult.result === null) {
```

Comment:
if `txResult.result === null` the error won't be catched

Reply:
https://github.com/GaloyMoney/galoy/pull/2072#discussion_r1058366152